src/main/java/org/candlepin/subscriptions/security/OptInChecker.java

```diff
@@ -61,7 +61,7 @@ public boolean checkAccess(Authentication authentication) {
      * the OptInRequiredException in the AccessDecisionVoter.vote method and then our own
      * AbstractAccessDecisionManager capable of catching that exception and rethrowing it after all
      * the other voters had been consulted. */
-    if (!accountConfigRepository.existsByOrgId(insightsUserPrincipal.getOrgId())) {
+    if (Boolean.FALSE.equals(orgConfigRepository.existsByOrgId(insightsUserPrincipal.getOrgId()))) {
```

Comment:
Is there any reason why `existsByOrgId` returns `Boolean` instead of the primitive `boolean`? 
Using the primitive type `boolean` would significantly make its usage easier because we don't need to check if it's null and we could simply do here:
```
if (!orgConfigRepository.existsByOrgId(insightsUserPrincipal.getOrgId())) {
```

Reply:
Thank you.  I have added this change.